spec/index.bs

```diff
@@ -14182,13 +14256,16 @@ is being composited into (e.g. an HTML page rendering, or a 2D canvas).
 
 During the normal course of operation of WebGPU, errors are raised via [$dispatch error$].
 
-After a device is [=lose the device|lost=] (described below), errors are no longer surfaced.
+After a device is [=lose the device|lost=] (described below), errors are no longer surfaced,
```

Comment:
This is an example of one of the "lost" references that could easily be updated:
```suggestion
After a device [=becomes lost=] (described below), errors are no longer surfaced,
```

Reply:
As above, I think it makes more sense for this to refer to [=lose the device=].